pkgs/applications/graphics/tesseract/tesseract5.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, autoconf-archive, pkg-config
+, leptonica, libpng, libtiff, icu, pango, opencl-headers, fetchpatch }:
+
+stdenv.mkDerivation rec {
+  pname = "tesseract";
+  version = "5.1.0";
+
+  src = fetchFromGitHub {
+    owner = "tesseract-ocr";
+    repo = "tesseract";
+    rev = version;
+    sha256 = "sha256-B1x3wxr9Sn2rsG8AHncPTEErhDo7YtpDRxfW9ZOPWoU=";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ pkg-config autoreconfHook autoconf-archive ];
```

Comment:
Might be nicer to have one dependency per line for easier git blame in the future.

Reply:
That’s done.